Source/JavaScriptCore/runtime/WeakMapPrototype.cpp

```diff
@@ -31,7 +31,7 @@
 
 namespace JSC {
 
-const ASCIILiteral WeakMapNonObjectKeyError { "Attempted to set a non-object key in a WeakMap"_s };
+const ASCIILiteral WeakMapInvalidKeyError { "Attempted to set neither an object nor a non-registered symbol key in a WeakMap"_s };
```

Comment:
This phrasing is pretty hard to read, let's make it more like:
```suggestion
const ASCIILiteral WeakMapInvalidKeyError { "WeakMap keys must be objects or non-registered symbols"_s };
```

Reply:
Looks good. Changed.